src/main/java/com/karimbouchareb/chromatographysimulator/ChromatographySimulatorApp.java

```diff
@@ -107,6 +107,166 @@ public static void main(String[] args) {
         launch();
```

Comment:
No space between line 104 and 105, javadoc should touch the main method. Also reference main as `{@link ChromatographySimulatorApp#main(String[])}`.

Reply:
Same with launch(), use proper link.